Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/batteryInput/index.tsx

```diff
@@ -1,4 +1,8 @@
-const BatteryInput = () => {
+interface BatteryInputProps {
+  onChangeHandler: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const BatteryInput = ({onChangeHandler} : BatteryInputProps) => {
```

Comment:
You should be controlling your inputs here. Let's say we have a function in App.tsx that runs setBattery(12), will the input field show 12 after?